chainreg/chainregistry.go

```diff
@@ -613,7 +625,19 @@ func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
 		// Use a query for our best block as a health check.
 		cc.HealthCheck = func() error {
 			_, _, err := cc.ChainSource.GetBestBlock()
-			return err
+			if err != nil {
+				return err
+			}
+
```

Comment:
Just tested this locally on regtest. I think we should skip the test for local networks, as it doesn't really give us much other than spamming the log.

```
			// On local test networks we usually don't have multiple
			// chain backend peers, so we can skip that test.
			if cfg.Bitcoin.SimNet || cfg.Bitcoin.RegTest {
				return nil
			}
```